charts/tekton-pipelines/values.yaml

```diff
@@ -65,22 +65,36 @@ buildpacks:
           resources: {}
           workingDir: $(resources.inputs.app.path)
           script: |
-            #!/bin/bash
+            #!/usr/bin/env bash
             set -Eeo pipefail
 
-            [ -f package.json ] && {
-              echo "install node.js dependencies"
-
+            if [[ ! -f package.json ]]; then
+              echo "No package.json found, stepping out"
+            else
+              echo "Installing node.js dependencies"
               # set cache
               npm config set cache $(workspaces.source.path)/.npm --global
-              npm ci
-
-              (npm run build:$(params.environment) && rm -rf node_modules && echo "remove node_modules dir") || (echo "unable to build $(params.environment) exit_code: $?" && exit 1)
+              if ! npm ci; then
```

Comment:
Misha - just convert all `ifs` to use the same style i.e `if [[ bool ]]`

Reply:
then test/merge and release as 0.1.35 into main

Reply:
Done